File: backend/Models/Transportation.cs
namespace OrtegaTourism.Models
{
    public class Transportation
    {
        public int Id { get; set; }
        public string Name { get; set; } = string.Empty;
        public string Type { get; set; } = string.Empty; // Bus, Taxi, Moto-taxi, etc.
        public string Description { get; set; } = string.Empty;
        public string Route { get; set; } = string.Empty;
        public string Schedule { get; set; } = string.Empty;
        public decimal Price { get; set; }
        public string Contact { get; set; } = string.Empty;
        public List<string> Services { get; set; } = new List<string>();
        public string ImageUrl { get; set; } = string.Empty;
    }
}

File: backend-laravel/app/Models/Transportation.php
<?php

namespace App\Models;

class Transportation
{
    public int $id = 0;
    public string $name = '';
    public string $type = '';
    public string $description = '';
    public string $route = '';
    public string $schedule = '';
    public float $price = 0;
    public string $contact = '';
    public array $services = [];
    public string $imageUrl = '';
    
    public function toArray(): array
    {
        return [
            'id' => $this->id,
            'name' => $this->name,
            'type' => $this->type,
            'description' => $this->description,
            'route' => $this->route,
            'schedule' => $this->schedule,
            'price' => $this->price,
            'contact' => $this->contact,
            'services' => $this->services,
            'imageUrl' => $this->imageUrl
        ];
    }
}
